catkin_home/src/devices/scripts/exampleShow.py

```diff
@@ -8,9 +8,10 @@
 
 
 def callback(data):
+    print("yeet")
     bridge = CvBridge()
-    cv_image = bridge.imgmsg_to_cv2(data, desired_encoding="passthrough")
-    cv2.imshow("frames",cv_image)
+    cv_image = bridge.imgmsg_to_cv2(data, desired_encoding="bgr8")
+    cv2.imshow("image",cv_image)
     if cv2.waitKey(1) & 0xFF == ord('q'):
```

Comment:
Question
The topic you are subscribed to called "frames" refers to which of the cameras' topics? Or does it refer to something else?

Reply:
That topic is published in imageCapture.py and it is possible to change the camera it is reading